### ISSUE TITLE
Why Assume the ASP Does Not Go Down?

### ISSUE THREAD

**Issue by KeithTimimi** [TashaARK/ARK-Invest-Tesla-Valuation-Model#14]
Both the Bull and Bear case assume an ASP of $48,250 thru 2023, which does not seem logical considering two facts: the price curve of EVs declines, following the decline of battery pack costs; and that in order to sell 3m or 1.7m vehicles, you will probably need to lower the sale price in order to expand the potential market.

For Tesla specifically, it is also Elon's stated aim to achieve a $35k base price. That is extremely challenging at current battery and operating costs, but will be achievable within the next 4 years as battery pack costs / SG&A% declines.

**Comment by jasonpiepmeier** [comment]
@TashaARK does this include Semi in pricing?  If so, how many Semi units are you assuming?  Does it include Roadsters?  If so, how many Roadster sales?  that could be pushing up ASP.  Would be interested in the average ASP of all Mercedes, BMW or Audi if you have that info.  I think the $48,250 sounds reasonable, as you will still have Model S, Model X, Roadster that are pushing prices up, while some base Model 3's are pushing the ASP down.  It's a balance, some consumers are going to get an S, X or Roadster, and some are going to get a Performance Model 3 with a higher ASP, and yes, some are going to buy a base Model 3 at $35k or below as prices drop.

**Comment by KeithTimimi** [comment]
I think that is a fair assumption for the Bull case Jason - but not for the Bear case. I think the Roadster launch date is still unknown. In a Bear scenario, it will be delayed, and pricing pressure will pull all the averages down.

**Comment by KeithTimimi** [comment]
Good point on the Semi though Jason, @TashaARK it might be worth having a separate section for that.
